model-engine/model_engine_server/inference/forwarding/forwarding.py

```diff
@@ -509,13 +509,22 @@ def _substitute_config_overrides(config: dict, config_overrides: List[str]) -> N
             raise ValueError(f"Error setting {key_path} to {value} in {config}") from e
 
 
+def _cast_value(value: Any) -> Any:
```

Comment:
does json.loads() work here or do we need something different? seems we're parsing nested python lists of integers

Reply:
it's not quite json.loads cause the values themselves don't need to be wrapped in strings. Right now i'm just trying to support ARGS=[/v1/chat/completions]